prisma/schema/user.prisma

```diff
@@ -0,0 +1,16 @@
+enum Role {
+  UG_TUTOR
+  REVIEWER
+  ADMIN
+  DEV
+}
+
+model User {
+  id              Int           @id @default(autoincrement())
+  admissionsCycle Int
+  loginId         String
```

Comment:
Why not just `login`?

Reply:
name changed in f9020b083dce1e93a9910b600573f7298c06b90b